invesalius/gui/task_navigator.py

```diff
@@ -63,6 +63,7 @@
 import invesalius.gui.dialogs as dlg
 import invesalius.project as prj
 from invesalius import utils
+from invesalius.net.pedal_connection import PedalConnection
```

Comment:
If pedal_connection uses optional modules is better wrap this import in a try except, something like this:

```python
HAS_PEDAL_CONNECTION = True
try:
    from invesalius.net.pedal_connection import PedalConnection
except ImportError:
   HAS_PEDAL_CONNECTION = False
```
Then You can check HAS_PEDAL_CONNECTION to know if PedalConnection was imported and if it's possible to create all GUI and variables.

Reply:
Good point, that kind of things needs to be done. It's done now -- I amended the previous commit to keep the number of commits a bit smaller. I also tested that it now works without the optional requirements installed and without the --use-pedal command line argument.